app/src/main/res/layout/revision_card_fragment.xml

```diff
@@ -28,9 +28,12 @@
         style="@style/Body"
         android:layout_marginStart="@dimen/revision_card_fragment_layout_margin_start"
         android:layout_marginTop="@dimen/revision_card_fragment_layout_text_margin_top"
-        android:layout_marginEnd="@dimen/revision_card_fragment_layout_margin_end" />
+        android:layout_marginEnd="@dimen/revision_card_fragment_layout_margin_end"
+        android:minWidth="48dp"
+        android:minHeight="48dp" />
 
       <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/navigation_card_container"
```

Comment:
Given that IDs are global, could we make this more specific?

Reply:
done